core/src/main/resources/schema/oracle/oracle-schema.sql

```diff
@@ -1,3 +1,24 @@
+ALTER SESSION SET ddl_lock_timeout = 15
+/
+
+DROP TABLE "journal" CASCADE CONSTRAINT
+/
+
+DROP TABLE "snapshot" CASCADE CONSTRAINT
+/
+
+DROP TABLE "deleted_to" CASCADE CONSTRAINT
+/
+
+DROP TRIGGER "ordering_seq_trigger"
+/
+
+DROP PROCEDURE "reset_sequence"
+/
+
+DROP SEQUENCE "ordering_seq"
+/
+
```

Comment:
I know this aligns the script to those for MySQL, Postgre, etc... where there is a DROP first and CREATE later, but this PR is also moving the `.sql` scripts to `main/` (from `test/`).

I'd rather we had (ideally idempotent) `.sql` scripts in `main/` that only created and then, somewhere else (e.g. `test/`) the scripts to drop.

Reply:
As a side effect of that split, we would be improving some cases (for example postgre-schema.sql) where there is:

```sql
DROP TABLE IF EXISTS public.journal;

CREATE TABLE IF NOT EXISTS public.journal (
...
);

DROP TABLE IF EXISTS public.snapshot;

CREATE TABLE IF NOT EXISTS public.snapshot (
...
```

Which looks a bit weird ("create table if not exists" immediately after "drop table"?).

Reply:
Ok to remove the `IF NOT EXISTS` from create table. 

Since the goal is to use it as a helper for building test, I do think it should drop and create by default. Users building tests do need to drop. If they don't want to drop the schema, they can use `applySchema` instead and pass a another file.

Reply:
I was thinking about what I just said here. 

Maybe we should indeed split the scripts and have: `drop`, `create`, `dropAndCreate` and `applyScript`. The later being the generic one.